app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -0,0 +1,41 @@
+.chat-box__container {
+  background-color: $color-white;
+  position: fixed;
+  top:30%;
+  left: 0;
+  padding: $baseline-unit*3;
+  z-index:9999;
+  border-radius: 0 4px 4px 0;
+  border: 2px solid $color-green-primary;
+  border-left-style: none;
+  -webkit-box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
+  -moz-box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
+  box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
+
+  @include respond-to($mq-xs) {
+    display: none;
+  }
+
+  @include respond-to($mq-l) {
+    display: block;
+  }
+}
+
+.chat-box__container:hover{
+  text-decoration: none;
+}
```

Comment:
Could you move this block nested inside the ``.chat-box__container`` class above like so:

```
.chat-box__container {
  ...

  &:hover {
    text-decoration: none;
  }
}
```